runtime/array.c

```diff
@@ -161,6 +155,9 @@ static value caml_array_unsafe_set_addr(value array, value index,value newval)
 }
 
 /* [ floatarray -> int -> float -> unit ] */
+/* [MM]: [caml_array_unsafe_set_addr] has a fence for enforcing the OCaml
+   memory model through its use of [caml_modify]. [caml_floatarray_unsafe_set]
+   will also need a similar fence in [Store_double_flat_field]. */
```

Comment:
Is this a TODO? I've looked at `Store_double_flat_field` and I don't see the fence.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/f6961db257c049a63a46838205186dc9a8d7a1ca